### DISCUSSION TITLE
Who should be able to vote on RFCs

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#2928] [Polls]
Looking into our Community, we have various groups of "shareholders":

* maintainer
* code contributors
* all contributors
* agencies / extension vendors / partners / hoster
* e-commerce companies using OpenMage

Now originally it was intended for RFCs that all of them get an equal voice, because all of them are equally important.
But I wonder if this is still what the community wants.

If you add something else as a comment, remember we have the upvote ⬆ (as known from StackOverflow)

**Comment by fballiano** [comment]
I think ecommerce companies can express their opinion but shouldn't vote since they're not actively participating. If they want to have a saying they could gift some manpower to the project. It seems fair.

**Reply by justinbeaty** [reply]
I originally voted `e-commerce companies using OpenMage (and above)` but changed to `All Contributors (and above)`, because I agree that an agency or company using OpenMage could easily devote some time to PRs.

**Comment by sreichel** [comment]
@Flyingmana maybe we should clarify first why who is listed as "partner"?

- Which contributor represents which partner?
- Is anyone listed as a partner without participating (or advertising) in the project?
